### ISSUE TITLE
loading the parameters

### ISSUE THREAD

**Issue by OfirKedem** [zoogzog/chexnet#11]
Hi zoogzog
I've been trying to run your code but it seems the DenseNet121 module has been changed.
meaning your parameters are saved under the name:
_module.densenet121.features.denseblock1.denselayer1.norm.1.bias_
------------------------------------------------------------------------------------^----------
while the module is telling me it's missing:
_module.densenet121.features.denseblock1.denselayer1.norm1.bias_

i believe that little dot there is messing everything up... 
do you have an idea how to solve it? loading an old DenseNet121 module or perhaps updating the parameters names? 

i think retraining the model can fix it but I'm trying to save time and avoid training for now.

thanks in advance,
a very desperate student  :)

**Comment by zoogzog** [comment]
If you want to use the trained model, I think the quickest solution would be to use the older version of the DenseNet implementation.

**Comment by zoukai214** [comment]
I met the same problem. Have you solved it? @OfirKedem

**Comment by OfirKedem** [comment]
Yes i have. You need to roll back torch and torchvision. Use this versions:
![image](https://user-images.githubusercontent.com/39023976/40246900-293438d4-5ad3-11e8-957b-05423c6f2fc1.png)

**Comment by dgrechka** [comment]
Spent some time trying to find torch v0.3.1 for windows. bad luck. It is not available anymore.
So pretrained weights are not usable.

**Comment by mshmoon** [comment]
This happend is because the high version pytorch define model keys as convX. or normX., but in the low version pytorch define  model keys as conv.X. or norm.X., they are defference, so you should define keys as the same name.
